scripts/partition_manager.py

```diff
@@ -546,6 +576,12 @@ def parse_args():
     parser.add_argument("--input-files", required=True, type=str, nargs="+",
                         help="List of paths to input yaml files. ")
 
+    parser.add_argument("--ram-size", required=True, type=int,
```

Comment:
Maybe the size should be in bytes, since RAM could have a size not divisible by 1024. Consider changing the flash-size as well to keep it consistent.

Reply:
Doesn't CONFIG_SRAM_SIZE being specified in kB already make this unsupported?

Reply:
Good point.

Still, if we for some reason want to partition subsections of RAM or FLASH, a minimum resolution of 1024 bytes seems arbitrary.

Reply:
fixed, see fixup commit